src/libraries/System.Formats.Tar/tests/TarReader/TarReader.TarEntry.ExtractToFile.Tests.Unix.cs

```diff
@@ -9,7 +9,7 @@ namespace System.Formats.Tar.Tests
 {
     public partial class TarReader_TarEntry_ExtractToFile_Tests : TarTestsBase
     {
-        [PlatformSpecific(TestPlatforms.AnyUnix & ~TestPlatforms.tvOS)] // https://github.com/dotnet/runtime/issues/68360
+        [PlatformSpecific(TestPlatforms.AnyUnix & ~TestPlatforms.tvOS & ~TestPlatforms.LinuxBionic)] // https://github.com/dotnet/runtime/issues/68360
         [ConditionalFact(nameof(IsUnixButNotSuperUser))]
         public void SpecialFile_Unelevated_Throws()
```

Comment:
@directhex any idea why this might fail on Linux Bionic but apparently not on Android?

Reply:
Not offhand